src/Symfony/Component/HttpFoundation/Session/Storage/Handler/NativeFileSessionHandler.php

```diff
@@ -23,7 +23,7 @@ class NativeFileSessionHandler extends NativeSessionHandler
      *                         Default null will leave setting as defined by PHP.
      *                         '/path', 'N;/path', or 'N;octal-mode;/path
      *
-     * @see http://php.net/session.configuration.php#ini.session.save-path for further details.
+     * @see https://www.php.net/manual/en/session.configuration.php#ini.session.save-path for further details.
```

Comment:
shorter URLs are nicer to me: https://php.net/manual/session.configuration.php#ini.session.save-path